TombEngine/Physics/Objects/CollisionMesh.cpp

```diff
@@ -58,7 +58,7 @@ namespace TEN::Physics
 		const auto& vertex1 = GetVertex1(vertices);
 		const auto& vertex2 = GetVertex2(vertices);
 
-		return Geometry::GetBoundingBox({ vertex0, vertex1, vertex2 });
+		return Geometry::GetAabb({ vertex0, vertex1, vertex2 });
```

Comment:
I don't fully understand why you are renaming `BoundingBox` to `Aabb` everywhere. First off, it removes clarity. Second, AABB is a very specific terminology which means that bounding box is axis-aligned in 3D space. But abstract bounding box which is _not_ in 3D space can't be axis-aligned, because there's no axis defined for it.